Makefile

```diff
@@ -29,13 +29,15 @@ BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-# BUNDLE_IMG defines the image:tag used for the bundle.
-# You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
-BUNDLE_IMG ?= controller-bundle:$(VERSION)
-
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
 
+# BUNDLE_IMG defines the image:tag used for the bundle.
+BUNDLE_IMG ?= controller-bundle:latest
+
+# The image tag given to the resulting catalog image (e.g. make catalog-build CATALOG_IMG=example.com/operator-catalog:v0.2.0).
+CATALOG_IMG ?= controller-catalog:latest
+
```

Comment:
I feel like we should stay consistent with using `$(VERSION)` so that our tags will match across the images.

Reply:
The current atlas operator uses the latest image version. It makes sense t change to use $VERSION as there 3 different images and this makes them consistent.